lib/attribute.js

```diff
@@ -0,0 +1,46 @@
+exports.getAttributeValue = function (prop) {
+  let X = prop.value
```

Comment:
best to not use capital letters for variable names

Reply:
that was from Karang's code, I can fix